spiketools/stats/shuffle.py

```diff
@@ -189,7 +189,7 @@ def shuffle_circular(spikes, shuffle_min=20000, n_shuffles=1000, random_state=No
     Parameters
     ----------
     spikes : 1d array
-        Spike times.
+        Spike times, in milliseconds.
     shuffle_min : int
         The minimum amount to rotate date, in terms of units of the spike train.
```

Comment:
I just noticed there is a type here ('date' which should be 'data'). I know this typo doesn't come from your changes here, but could you fix it in this PR?

Reply:
Good catch! Just fixed it.